service/src/main/java/aero/minova/cas/controller/SqlViewController.java

```diff
@@ -160,15 +157,11 @@ public Table getIndexView(@RequestBody Table inputTable) throws Exception {
 				limit = inputMetaData.getLimited();
 			}
 
-			String viewQuery;
 			// POSTGRE SQL verwendet RowCount als Funktion, wesewegen es nicht so genutzt werden kann, wie wir es bei der pagingWithSeek-Methode verwenden.
 			// Deshalb verwenden wir stattdessen die prepareViewString-Methode, welche minimal langsamer ist.
-			if (databaseKind.equals("postgresql")) {
-				viewQuery = prepareViewString(inputTable, false, 0, authoritiesForThisTable);
-			} else {
-				// Wir setzten für Page 1 und für limited 0 ein, damit wir alle Ergebnisse bekommen. Die Menge wird später begrenzt.
-				viewQuery = pagingWithSeek(inputTable, false, 0, false, 1, authoritiesForThisTable);
```

Comment:
Bitte für pagingWithSeek JavaDoc erstellen und kommentieren, warum es nicht für `data/index` verwendet wird (könnte sein, dass wir in Zukunft es vielleicht brauchen werden).